Makefile.am

```diff
@@ -1,3 +1,7 @@
+# DOXYGEN SUPPORT
+# include $(top_srcdir)/aminclude.am
```

Comment:
left over from an earlier version of the ax_prog_doxygen macro that was splitting the features and the makefile rules. will remove.